entries/x/xapi.us.json

```
@@ -0,0 +1,12 @@
+{
+  "XAPI": {
+    "domain": "xapi.us",
+    "tfa": [
+      "totp"
+    ],
+    "keywords": [
+      "developer",
+      "gaming"
```

Whilst the service does seem to integrate with a gaming service I wouldn't classify XAPI as a gaming service.
```suggestion
      "developer"
```